graph/scaling.py

```diff
@@ -94,6 +102,13 @@ def scaling_graph(args, output_dir, job: str, traces_name: list) -> int:
                     outfile = f"scaling_watt_{clean_perf}_{bench.get_title_engine_name().replace(' ','_')}"
                     y_label = "Watts"
                     y_source = aggregated_watt
+                elif "cpu_clock" in graph_type:
```

Comment:
why not use == instead of in here? Since these are magic values.

Reply:
I kept the same style as the previous ones, I could make a commit to clean all them at once if you want